Automattic/wp-calypso
# Calypso: Add new entry point

#### Changes proposed in this Pull Request
* This change will make it possible to have a separate section for the jetpack cloud.
By allowing this current section to be replaced by something jetpack cloud specific. 

#### Testing instructions

Run `npm start`.
Visit calypso.localhost:3000 does the page redirect you as expected to the reader or the home section of a site. 

I am not sure what is the best way to test this.
